docs-src/index.md

```diff
@@ -1,36 +1,52 @@
 # Introduction
 
-Service APIs is the evolution of Kubernetes APIs that relate to `Services`,
-e.g. `Ingress`. The project is part of Kubernetes, working under
-[SIG-NETWORK][sig-network]. Use the following resources to learn more about
-Service APIs.
+Service APIs is an open source project, working under
+[SIG-NETWORK][sig-network], that is evolving service networking APIs within the Kubernetes ecosystem. Service APIs are the APIs you use to expose your Kubernetes applications - Services, Ingress, and more. The Gateway API is the primary API being developed to unify service networking under a core set of networking APIs.
+
+### What is the Gateway API?
```

Comment:
I don't think we should use `Gateway API` as a name here. Service APIs are broader than that resource.

Reply:
+1 here.
I found this term being used GCP's repo as well: https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/master/gateway

There is already some confusion brewing: https://github.com/kubernetes-sigs/service-apis/issues/332#issuecomment-719401062